tests/utils/utils.go

```diff
@@ -263,7 +264,7 @@ func SendERC20Source(
 	input erc20source.SendTokensInput,
 	amount *big.Int,
 	senderKey *ecdsa.PrivateKey,
-) (*types.Receipt, *big.Int) {
+) (*types.Receipt, *big.Int, teleportermessenger.TeleporterMessage) {
```

Comment:
Since most invocations of this function don't use the `teleportermessenger.TeleporterMessage` return value, I think we should not return it, and just parse it from the receipt in the cases where we need it.

Reply:
addressed in https://github.com/ava-labs/teleporter-token-bridge/pull/85/commits/c530cd967c925a05eab602d7eb4f64c0d4a31705